app/views/directives/edit-environment-from.html

```diff
@@ -0,0 +1,94 @@
+<ng-form name="forms.editEnvironmentFrom" novalidate>
+  <div
+    ng-if="$ctrl.showHeader"
+    class="key-value-editor-entry key-value-editor-entry-header">
+    <div class="form-group key-value-editor-header value-header">
+      <div class="input-group">
```

Comment:
Why wrap in a `form-group` and `input-group`?

Reply:
Part of the classes copied over from the variables component (heads up there's may be some clean up there too), also part of the CSS clean up we've been going through, lots of extras to remove.

Reply:
OK to leave it's reusing the styles and needed